lib/auth/api.go

```diff
@@ -201,6 +201,9 @@ func (w *Wrapper) CreateAuditStream(ctx context.Context, sid session.ID) (events
 	return w.NoCache.CreateAuditStream(ctx, sid)
 }
 
+func (w *Wrapper) GetUploadMetadata(sid session.ID) events.UploadMetadata {
```

Comment:
This is not required - only `ProtoStreamer` should implement `UploadMetadataGetter`.